ResetButton.gd

```diff
@@ -0,0 +1,16 @@
+extends Button
+
+# Declare member variables here. Examples:
+# var a = 2
+# var b = "text"
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	pass # Replace with function body.
+
+func _on_click():
+	get_tree().change_scene("res://Board.tscn")
```

Comment:
I would rather create an script for `ControlPanel` (`ControlPanel.gd`) and move this function there